admin-dev/themes/new-theme/js/pages/attribute/form/attribute-form-map.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+/**
+ * Defines all selectors that are used in attribute address add/edit form.
+ */
+export default {
+  attributeShopAssociationInput: '#attribute__shop_association',
```

Comment:
Shouldn't it be this instead?
```suggestion
  attributeShopAssociationInput: '#attribute_shop_association',
```

Reply:
Still this unanswered question, form IDs generated by symfony follow the property path in the form basically, so I don't really understand why there is an empty space in the middle of this generated ID? 🤔

Id the ID is incorrect then the component initialization of ChoiceTree can't work as expected